Assets/_Scripts/Hero.cs

```diff
@@ -4,15 +4,54 @@
 
 public class Hero : MonoBehaviour
 {
-    // Start is called before the first frame update
-    void Start()
+    static public Hero S;
+
+    [Header("Set in Inspector")]
+
+    //These fields control the movement of the ship
+
+    public float speed = 30;
+
+    public float rollMult = -45;
+
+    public float pitchMult = 30;
+
+    [Header("Set Dynamically")]
+
+    public float shieldLevel = 1;
+
+    void Awake()
     {
-        
+        if (S == null)
+        {
+            S = this;
+        }
+        else
+        {
+            Debug.LogError("Hero.Awake() - Attempted to assign second Hero.S!");
+        }
     }
 
-    // Update is called once per frame
     void Update()
     {
-        
+        //Pull in information from the Input class
+
+        float xAxis = Input.GetAxis("Horizontal");
+
+        float yAxis = Input.GetAxis("Vertical");
+
```

Comment:
Just keep in mind - this is the old Input Manager method. We may want to refactor and use Input System